scripts/install_debs_cmake.sh

```diff
@@ -19,8 +19,27 @@ set -o verbose
 
 # Install the required libraries that are available as debs.
 sudo apt-get update
+
+# Install CMake 3.2 for Ubuntu Trusty and Debian Jessie.
+sudo apt-get install lsb-release -y
+if [[ "$(lsb_release -sc)" = "trusty" ]]
+then
+  sudo apt-get install python-software-properties apt-file -y
+  sudo apt-file update
+  sudo apt-get install software-properties-common -y
+  sudo add-apt-repository ppa:george-edison55/cmake-3.x
```

Comment:
no need to install personal (untrusted) packages,
just install `cmake3` https://packages.ubuntu.com/en/trusty/cmake3

Reply:
oh, thanks, done